components/ChatFooter.vue

```diff
@@ -133,16 +133,21 @@
             </b-button>
           </div>
         </span>
-        <b-button variant="primary" class="float-end ml-2 mr-2" @click="send">
-          Send&nbsp;
-          <v-icon
-            v-if="sending"
-            icon="sync"
-            class="fa-spin"
-            title="Sending..."
-          />
-          <v-icon v-else icon="angle-double-right" title="Send" />
-        </b-button>
+        <spin-button
```

Comment:
You've got `SpinButton` in some places and `spin-button` in others.  I prefer `SpinButton` - please make consistentn.